aten/src/ATen/native/native_functions.yaml

```diff
@@ -320,6 +320,16 @@
 
 - func: cat_out(Tensor result, TensorList tensors, int64_t dim=0) -> Tensor
 
+- func: cat(TensorList tensors, int64_t dim, Scalar pad_value) -> Tensor
+  python_default_init:
+    dim: 0
+    pad_value: NAN
+
+- func: cat_out(Tensor result, TensorList tensors, int64_t dim, Scalar pad_value) -> Tensor
+  python_default_init:
+    dim: 0
+    pad_value: NAN
```

Comment:
it would be good to comment on the hack here.

Reply:
What should I comment?